tensorflow/core/framework/tensor.cc

```diff
@@ -677,14 +677,42 @@ bool Tensor::CanUseDMA() const {
 #undef CASE
 
 namespace {
+//from left dim to right dim , recursive print
```

Comment:
Here and below, please abide to google C++ style guide, to wit:
Need one space after '//'
Comments should be a full sentence, including punctuation, for instance "Loop every element of one dim."